db.php

```diff
@@ -1157,10 +1167,7 @@ public function db_version( $dbh_or_table = false ) {
 			$dbh = $this->db_connect( "SELECT FROM $dbh_or_table $this->users" );
```

Comment:
This ends up being `SELECT FROM false $this->users`, is that intended? Feels a bit strange, but it does work as it gets through `db_connect()` with a global read connection which is what we want I suppose.